integration

```diff
@@ -148,6 +148,7 @@
   "chises/ha-oilfox",
   "Chouffy/home_assistant_libratone_zipp",
   "Chouffy/home_assistant_tgtg",
+  "chris-mc1/qBittorrent-hass",
```

Comment:
<https://github.com/chris-mc1/qBittorrent-hass/blob/0.0.6/hacs.json#L3>
`my_super_awesome_thing.js` ??
This is not valid..